src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  transition: transform $card-transition-duration;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
```

Comment:
redundant here

But generally, it's better to set for each element on the page (use `*` selector)